test/test_pubsub.py

```diff
@@ -0,0 +1,58 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+#
+# Copyright (C) 2021 Michal Galka
```

Comment:
```suggestion
# Copyright (C) 2021 Collabora Limited
```

Reply:
Nice catch. I also corrected the year as we're having 2022.